tests/unit/amo/components/TestSectionLinks.js

```diff
@@ -24,14 +34,9 @@ describe('SectionLinks Component', () => {
     expect(root.find('.SectionLinks-link').length).toEqual(3);
   });
 
-  it('renders no active links without context', () => {
-    const root = render({ viewContext: null });
```

Comment:
Interestingly, this scenario is no longer possible to test because it's not possible in the real app!